bin/templates/cordova/lib/emulator.js

```diff
@@ -238,13 +242,13 @@ module.exports.wait_for_emulator = function(num_running) {
  */
 module.exports.wait_for_boot = function(emulator_id) {
     var self = this;
-    return exec('adb -s ' + emulator_id + ' shell getprop init.svc.bootanim', os.tmpdir())
+    return exec('adb -s ' + emulator_id + ' shell ps', os.tmpdir())
     .then(function(output) {
-        if (output.match(/stopped/)) {
+        if (output.match(/android\.process\.acore/)) {
             return;
         } else {
             process.stdout.write('.');
-            return Q.delay(3000).then(function() {
+            return Q.delay(7000).then(function() {
```

Comment:
From my observations, `shell ps` (getting the list of all processes) is more time consuming than `shell getprop` (getting one property) so I increased the polling time.
This is only my feeling, we can safely return it back to 3s I think.